src/interpreter/fvm3.rs

```diff
@@ -105,7 +105,8 @@ impl<DB: Blockstore> Externs for ForestExterns<DB> {}
 
 impl<DB> Chain for ForestExterns<DB> {
     fn get_tipset_cid(&self, epoch: ChainEpoch) -> anyhow::Result<Cid> {
-        (self.get_tsk)(epoch)?.cid()
+        // (self.get_tsk)(epoch)?.cid()
+        Ok(Default::default())
```

Comment:
Great for testing. Not great for production. Too bad we don't have a regression test for this.